zaproxy/src/org/zaproxy/zap/extension/attacksurfacedetector/ZapPropertiesManager.java

```diff
@@ -130,23 +130,15 @@ public String getTargetPath()
     public boolean getAutoSpider()
     {
         String autoSpider = getProperties().getProperty(AUTO_SPIDER_KEY);
-        boolean shouldSpider = false;
-        if (autoSpider == null)
-            shouldSpider = false;
-        if (autoSpider.equalsIgnoreCase("true"))
-            shouldSpider = true;
+        boolean shouldSpider = Boolean.parseBoolean(autoSpider);
         logger.info("returning autospider " + autoSpider + "(should spider? " + shouldSpider + ")");
         return shouldSpider;
     }
 
     public boolean getUseHttps()
     {
         String useHttps = getProperties().getProperty(HTTPS_KEY);
-        boolean shouldUseHttps = false;
-        if (useHttps == null)
-            shouldUseHttps = false;
-        if (useHttps.equalsIgnoreCase("true"))
-            shouldUseHttps = true;
+        boolean shouldUseHttps = Boolean.parseBoolean(useHttps);
         logger.info("returning useHttps " + useHttps + "(should use https? " + shouldUseHttps +")");
```

Comment:
This will need to be modified as well.

Reply:
Do you mean to print other thing than "null"?

Reply:
My bad, (I should stop doing code reviews on my phone), saw the bar removal above and that it was still used here....but u did reassign it, so really it’s fine.